spec/features/home_spec.rb

```diff
@@ -30,7 +30,9 @@
 
     page.scroll_to(:bottom)
 
-    wait_for { Event.count }.to eq(event_count_before + 1)
+    Prosopite.pause do
+      wait_for { Event.count }.to eq(event_count_before + 1)
+    end
```

Comment:
I find it ever so slightly annoying that Prosopite catches "N + 1" queries in specs. I wouldn't even call this a real "N + 1" query; it's just that the same query happens to be made multiple times. But it's not a big negative, and I guess maybe that's just the price to be paid for Prosopite's otherwise useful N + 1 query detection.